soh/src/code/z_camera.c

```diff
@@ -1483,7 +1483,7 @@ s32 Camera_Free(Camera* camera) {
 
     sCameraInterfaceFlags = 1;
 
-    camera->animState = 1;
+    camera->animState = 0;
```

Comment:
could you add a comment to this line explaining why this is being set to zero?

Reply:
This was @MelonSpeedruns 's suggestion, so he can probably offer more insight on that. It fixes the bugs but causes the issue which is fixed by removing the other two lines mentioned above.